manga109api/manga109api.py

```diff
@@ -17,22 +21,77 @@ def __init__(self, root_dir):
         with (self.root_dir / "books.txt").open("rt", encoding='utf-8') as f:
             self.books = [line.rstrip() for line in f]
 
-    def get_annotation(self, book, annotation_type="annotations"):
+
+    def get_annotation(self, book, annotation_type="annotations", separate_by_tag=True):
```

Comment:
Since we use `annotation_types = ["frame", "face", "body", "text"]` above, 
I guess we should change `annotation_type` to `annotation_dir` or something here.

And thanks to `separate_by_tag`, we can get the same result as before. Nice :D

Reply:
> I guess we should change `annotation_type` to `annotation_dir` or something here.

Oh, sorry I didn't notice that, in fact I was overloading `annotation_type` in a `for` block... Since we have the option separate_by_"tag" , I guess I should change `annotation_types = ["frame", "face", "body", "text"]` to `annotation_tags = ["frame", "face", "body", "text"]`.

> And thanks to `separate_by_tag`, we can get the same result as before. Nice :D

Thanks, and you're welcome! After writing some filtering code, I thought it was better in a lot of ways to have a mode to return the original format.